getnacelle/nacelle-js
# fix: broken dependencies

## Why are these changes introduced?

Addresses [ENG-11205](https://nacelle.atlassian.net/browse/ENG-11205) 

> Ensure that `nacelle-js` monorepo dependencies can be installed and bootstrapped

## What is this pull request doing?

This PR fixes dependencies in `examples/shopify-accounts` so that packages can be bootstrapped at the monorepo root.

It also updates Storefront SDK / Commerce Queries Plugin lockfiles.

## How to Test



1. Check out the `ENG-11205/fix-monorepo-dependencies` branch
2. `npm i && npm run bootstrap` - there should be no errors when installing or bootstrapping, and there should be no uncommitted changes

## Checklist


[ENG-11205]: https://nacelle.atlassian.net/browse/ENG-11205?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
